receiver/prometheusreceiver/internal/transaction.go

```diff
@@ -159,14 +156,6 @@ func (tr *transaction) Commit() error {
 		return err
 	}
 
-	if tr.metricBuilder.hasInternalMetric {
-		m := ochttp.ClientRoundtripLatency.M(tr.metricBuilder.scrapeLatencyMs)
```

Comment:
Removed redundant OC metric and related code since it must be registered via OpenCensus view to be used which we don't do in the collector.